autocertkit/utils.py

```diff
@@ -65,6 +65,7 @@
 
 # Capability Tags
 REQ_CAP = "REQ"
+REQ_CAP_MULTICAST = "REQ_MULTICAST"
```

Comment:
REQ_CAP is for 'required capability', meaning mandatory to pass to get certified. REQ_CAP_MULTICAST does not have such handle, hence I do not know implication of this.
But Is this really what you want? Tests passed without 'required' flag will still appear on results.txt with listed tests. Isn't is enough?